.changeset/brown-tigers-nail.md

```diff
@@ -0,0 +1,6 @@
+---
+"@graphiql/react": patch
+"graphiql": patch
+---
+
+fix: use the default query on adding a new tab
```

Comment:
```suggestion
fix: set query editor to `defaultQuery` while adding a new tab or GraphiQL's default query

```graphql
# Welcome to GraphiQL
#
# GraphiQL is an in-browser tool for writing, validating, and
# testing GraphQL queries.

....
```

```